packages/babel-plugin/src/visitors/stylex-create/parse-stylex-create-arg.js

```diff
@@ -72,9 +72,9 @@ export function evaluateStyleXCreateArg(
     const allParams: Array<
       NodePath<t.Identifier | t.SpreadElement | t.Pattern>,
     > = fnPath.get('params');
-    const params: Array<t.Identifier> = allParams
-      .filter((param): param is NodePath<t.Identifier> =>
-        pathUtils.isIdentifier(param),
+    const params: Array<t.Identifier | t.AssignmentPattern> = allParams
+      .filter((param): param is NodePath<t.Identifier | t.AssignmentPattern> =>
+        pathUtils.isIdentifier(param) || pathUtils.isAssignmentPattern(param),
```

Comment:
We need an additional check that the lhs of the assignment pattern is an identifier.